.github/workflows/build.yml

```diff
@@ -39,6 +39,14 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Paths filter
+        uses: ./.github/actions/paths-filter
```

Comment:
Is it possible to remove this 3rd party action?

Reply:
Then we need to implement a similar function ourselves, and I tried to find github action support for a similar function: trigger step by file modification path, but not found. I can't find the environment variable that indicates the file modification.